source/docs/software/advanced-controls/system-identification/viewing-diagnostics.rst

```diff
@@ -9,7 +9,7 @@ There are three numerical accuracy metrics that are computed with this tool: acc
 .. image:: images/viewing-metrics.png
    :alt: Analysis accuracy metrics
 
-The acceleration r-squared is the fraction of the variance in measured acceleration (used as the independent variable in the SysId regression) explained by the linear model.  This can be quite variable, because acceleration is very susceptible to system noise.  Assuming the other fit metrics are acceptable, values near ``1`` indicate an "ideal" mechanism with few disturbances, while values near ``0`` indicate a noisy mechanism with substantial physical vibrations/losses.
+The acceleration r-squared is the fraction of the variance in measured acceleration (used as the independent variable in the SysId regression) explained by the linear model.  This can be quite variable, because acceleration is very susceptible to system noise.  Mechanisms tend to vibrate quite a bit, so this value rarely goes above ``0.5``, even on very good datasets. If the acceleration r-squared goes below around ``0.2``, the kA gain will be of dubious quality and the mechanism vibration should be reduced if possible.
```

Comment:
I think we should add a sentence here that other results can still be valid.

Reply:
We also need to update all of the images on this page with the new layout.

Reply:
We need to update all of the images in "Analyzing Data" and refactor the text to work with the new boxes.  This could be a new PR if that is too overwhelming.

Reply:
I would also like to replace all of the instances in the docs with SysID to SysId.

Reply:
I think it's within scope for this PR definitely, my main considered tradeoff was "completeness" of the docs vs timescale for getting it out (thus deferral of the tutorial).